pages/index.js

```diff
@@ -1,5 +1,30 @@
+import TopicBox from "../components/organisms/TopicBox";
 import Layout from "../components/layout";
+import userSwr from "swr";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());
 
 export default function Home() {
-  return <Layout home></Layout>;
+  const { data, error } = userSwr("/api/topics", fetcher);
```

Comment:
I didn't notice this before, but this is kind of interesting. Grab a list of topics from an in-app API, right?

Reply:
yep!  it'll be a great way to mock this all out in the meantime... i'm going to add the functionality to get the subtopic links as one of my next steps